clang/test/Driver/sycl-offload-intelfpga.cpp

```diff
@@ -171,6 +171,16 @@
 // RUN:  | FileCheck -DOUTDIR=%t_dir -check-prefix=CHK-FPGA-REPORT-OPT %s
 // CHK-FPGA-REPORT-OPT: aoc{{.*}} "-sycl" {{.*}} "-output-report-folder=[[OUTDIR]]{{/|\\\\}}file.prj"
 
+/// -fintelfpga output report file from dir/source
```

Comment:
Will this test be run on all platforms?

Reply:
That is the expectation, but currently is not true reality.  There are currently adjustments that are needed to have this test run on a Windows machine - so the test is run but is expected to fail.  We at least exercise what would be Windows like paths by running clang_cl on the linux host.